src/GenericExtractorJob.php

```diff
@@ -106,9 +106,8 @@ public function run(): void
 
         $parentId = $this->getParentId();
 
-        /** @var RestRequest $request */
         $request = $this->firstPage($this->config);
-        while ($request !== false) {
+        while ($request !== null) {
```

Comment:
`Scroller` vracia uz `null` ak nie je ziadna dalsia stranka, tu to nebolo fixnute.